src/screens/ContactScreen/index.js

```diff
@@ -212,11 +274,13 @@ const ContactScreen = ({navigation}) => {
         // onPress={handleSearch}
       />
 
-      <View>
-        {selectedUsers && (
-          <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
-        )}
-      </View>
+      {!isAnon && (
+        <View>
+          {selectedUsers && (
+            <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+          )}
+        </View>
+      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ContactPreview` component is now conditionally rendered based on the `isAnon` state variable. This is a good practice as it allows the component to be displayed only when necessary, improving performance.
```diff
+ {!isAnon && (
+   <View>
+     {selectedUsers && (
+       <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+     )}
+   </View>
+ )}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->